CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## Unreleased
 
+### Fixed
+
+- Fix bug that it doesn't guard `#destroy` on activerecord 5.2.
```

Comment:
This bug affects `activerecord>=5.0`

Reply:
I think it doesn't matter on Rails 5.0 and 5.1 because the all tests passed on them at https://github.com/cookpad/guard_against_physical_delete/pull/29.

My understanding is like this:

- In Rails 5.0 and 5.1, `#delete`, `#destroy`, and `#destroy!` calls `.delete_all`, and this is supposed to be guared by `GuardAgainstPhysicalDelete::Relation::MethodOverrides#delete_all`.
- In Rails 5.2, `#delete`, `#destroy`, and `#destroy!` no longer calls `.delete_all`, as you mentioned at https://github.com/cookpad/guard_against_physical_delete/commit/d3a54e3a33ce63ab29983d5f3ed3fa9c30f85e8d#diff-27bfeb241dc2e1fd345106dfa60c6b866d09590271b8ca30780a6686d7d040a0R15, so we must guard them in some other way.

Reply:
My bad, you are right. The reason I thought that way was because `User.last.destroy` did not raise in Rails console of cookpad_all (Rails 5.0.7.1), but actually `User` is _not_ logically deletable.